static/src/stylesheets/module/_discussion.scss

```diff
@@ -840,13 +840,27 @@ $comment-recommend-button-size: 19px;
 .d-comment__actions {
     position: absolute;
     bottom: $gs-baseline;
+    opacity: .7;
     .discussion--closed & {
-        display: none;
+        .d-comment__action--reply, .d-comment__action-separator, .d-comment__action-pick {
+            display: none;
+        }
+        .sharing-component {
+            margin: 0 !important;
+
+            .sharing-text .inline-icon {
+                padding-left: 0 !important;
```

Comment:
This is also pretty heavily nested, we should try and keep CSS flat, so styling on a new class is preferable